lib/re3data/getMetrics.py

```diff
@@ -0,0 +1,54 @@
+import urllib.request
+import json
+
+base_url = 'https://www.re3data.org/metrics/data/'
+metrics = ['aidSystems',
+           'apis',
+           'certificates',
+           'contentTypes',
+           'dataAccess',
+           'dataAccessRestrictions',
+           'databaseAccess',
+           'databaseAccessRestrictions',
+           'databaseLicenses',
+           'dataLicenses',
+           'dataUploads',
+           'dataUploadRestrictions',
+           'enhancedPublication',
+           'institutionCountry',
+           'responsibilityTypes',
+           'institutionType',
+           'keywords',
+           'metadataStandards',
+           'pidSystems',
+           'providerTypes',
+           'qualityManagement',
+           'repositoryLanguages',
+           'software',
+           'subjects',
+           'syndications',
+           'types',
+           'versioning']
+
+f = open('re3data-metrics.json', 'w')
+f.write('{')
```

Comment:
Huh... seems a bit hacky, but if it works... you know you can easily serialise a Python dictionary to JSON?